app-editors/vscode/vscode-1.56.0.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop pax-utils xdg
+
+DESCRIPTION="Multiplatform Visual Studio Code from Microsoft"
+HOMEPAGE="https://code.visualstudio.com"
+SRC_URI="
+	amd64? ( https://update.code.visualstudio.com/${PV}/linux-x64/stable -> ${P}-amd64.tar.gz )
+	arm? ( https://update.code.visualstudio.com/${PV}/linux-armhf/stable -> ${P}-arm.tar.gz )
+	arm64? ( https://update.code.visualstudio.com/${PV}/linux-arm64/stable -> ${P}-arm64.tar.gz )
+"
+
+RESTRICT="mirror strip bindist"
+
+LICENSE="Microsoft-VSCode"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~arm ~arm64"
+IUSE=""
+
+RDEPEND="
+	app-accessibility/at-spi2-atk
+	app-crypt/libsecret[crypt]
+	dev-libs/nss
+	media-libs/alsa-lib
+	media-libs/libpng:0/16
+	x11-libs/cairo
+	x11-libs/gtk+:3
+	x11-libs/libnotify
+	x11-libs/libxkbcommon
+	x11-libs/libxkbfile
+	x11-libs/libXScrnSaver
+	x11-libs/libXtst
+	x11-libs/pango
+"
+
+QA_PRESTRIPPED="*"
+
+QA_PREBUILT="*"
```

Comment:
The intial EBUILD actually had a list there but then tinderbox detected some more files that haven't been specified. So I changed that to just include everything. Since everything is prestripped and prebuilt, are there some benefits to write the files one by one ?

Reply:
Maybe just have `QA_PREBUILT="/opt/vscode/*"`? Not much different than `"*"` given almost everything is installed there, but I kinda like it limited to /opt. This will still catch sub-directories like resources/app's binaries.

Also no need for `QA_PRESTRIPPED` if you have `QA_PREBUILT`, prebuilt cause to skip the majority of QA_*, including strip.

Reply:
@juippis what do you think of @ionenwks suggestion ? Is it a good "compromise" ?

Reply:
Yeah it's a bit better at least. I like the list to see what's being bundled etc.

Reply:
Got it, I'll add that then

Reply:
Done!